### ISSUE TITLE
User List

### ISSUE THREAD

**Issue by wrathagom** [samtecspg/articulate#853]
Add a list of users with the ability to create and delete users.

**Comment by broham89** [comment]
- Would you add people though email and/or articulate generated link? 
- Would this add to people who already downloaded articulate and use the interface? Or only to the chat testing?

**Comment by wrathagom** [comment]
I meant an add button with a form so the admin could manually add them. So far we only have one user list and it is server access. So it adds people who have access to that instance of Articulate.

**Comment by wrathagom** [comment]
There is a signup form, so people can sign-up themselves if that setting is turned on.

**Comment by broham89** [comment]
What is the parameter's title for the inputs? 
- Username?
- email?

**Comment by broham89** [comment]
![User list - page one](https://user-images.githubusercontent.com/11262149/63466057-0ac61380-c428-11e9-8445-2a3d02ce107a.png)

**Comment by broham89** [comment]
![User list - page 2](https://user-images.githubusercontent.com/11262149/63466874-cfc4df80-c429-11e9-87ca-c707c09a5b1d.png)

**Comment by wrathagom** [comment]
@broham89 I don't think that should be the add user button, but rather a button to the users page. Then there is a page that lists all of the users?

**Comment by broham89** [comment]
![User list - page one](https://user-images.githubusercontent.com/11262149/63533210-ee7eb100-c4d1-11e9-9abd-80db92f3b753.png)
![User list - page 2](https://user-images.githubusercontent.com/11262149/63533220-f179a180-c4d1-11e9-8cc4-b8f45875ec6c.png)
![User list - page 3](https://user-images.githubusercontent.com/11262149/63533227-f3dbfb80-c4d1-11e9-9f85-a537b36367a8.png)
[user-list-icon.svg.zip](https://github.com/samtecspg/articulate/files/3531094/user-list-icon.svg.zip)

**Comment by broham89** [comment]
![User list - page one](https://user-images.githubusercontent.com/11262149/63533348-36053d00-c4d2-11e9-8c16-583709cf0d19.png)
![User list - page 3](https://user-images.githubusercontent.com/11262149/63533363-42899580-c4d2-11e9-9dae-1a780fbd625b.png)
[user-list-icon.svg.zip](https://github.com/samtecspg/articulate/files/3531097/user-list-icon.svg.zip)
